docs/source/intro.rst

```diff
@@ -31,26 +31,25 @@ and unnoticed precision lost for identifier is what we want to avoid.
 Authentication
 ''''''''''''''
 
-Authentication to the Google ``BigQuery`` service is via ``OAuth 2.0``.
-Is possible to authenticate with either user account credentials or service account credentials.
+Authentication to the Google ``BigQuery`` service via ``OAuth 2.0``
+is possible to authenticate with either user account credentials or service account credentials.
 
-Authenticating with user account credentials is as simple as following the prompts in a browser window
-which will be automatically opened for you. You will be authenticated to the specified
-``BigQuery`` account using the product name ``pandas GBQ``. It is only possible on local host.
+Authentication via user account credentials is as simple as following the prompts in a browser window
+which automatically open for you. You authenticate to the specified
+``BigQuery`` account using the product name ``pandas GBQ``, but only possible on local host.
```

Comment:
Please remove the following text 

> , but only possible on local host

We support authenticating on both local and remote via the `auth_local_webserver` argument of `read_gbq`